dice.js

```diff
@@ -0,0 +1,16 @@
+const rollDice = function (){
+    const Dice1 = math.floor(math.random()*(6)+1);
```

Comment:
Javascript is case sensitive, you should  write `Math.random()` and `Math.floor()` with uppercase `M`.